mycore.org/content/de/documentation/basics/basics_mcr_ocfl.md

```diff
@@ -0,0 +1,312 @@
+
+<b class="text-warning">Es ist zu beachten, dass das MyCoRe Storage Layout keinem Standard entspricht und daher womöglich von keinen externen Tools nativ unterstützt wird.</b>
+
+Im Vergleich zu dem *0003-hash-and-id-n-tuple-storage-layout* ist das *mycore-storage-layout* besser ohne externe Tools zu navigieren. Es ist zu beachten, dass trotzdem keine Daten direkt verändert werden dürfen, da es sonst zu Fehlern bei der Validierung kommen kann.
+
+In der Performance in MyCoRe ist das *mycore-storage-layout* schneller als das *0003-hash-and-id-n-tuple-storage-layout*, da nicht erst bei jedem Dateizugriff die Hashwerte des Dateinamens generiert werden müssen.
+
+Link zur Spezifikation: [mycore-storage-layout.md <sup><i class="fas fa-external-link-alt"></i></sup>](https://github.com/MyCoRe-Org/mycore/blob/2021.06.x/mycore-ocfl/src/main/resources/ocfl-specs/mycore-storage-layout.md)
+
+#### Ordnerstruktur
+
+```shell {linenos=table}
+[storage root]
+├── 0=ocfl_1.0
+├── extensions
+│   └── mycore-storage-layout
+│       └── config.json
+├── mycore-storage-layout.md
+├── ocfl_1.0.txt
+├── ocfl_extensions_1.0.md
+├── ocfl_layout.json
+├── mcrderivate
+│   └── Project
+│       └── derivate
+│           ├── 0000
+│           │   └── 01
+│           │       ├── Project_derivate_00000101
+│           │       │   └── ... [object root]
+│           │       ├── Project_derivate_00000109
+│           │       │   └── ... [object root]
+│           │       └── Project_derivate_00000110
+│           │           └── ... [object root]
+│           └── 1234
+│               └── 56
+│                   └── Project_derivate_12345678
+│                       └── ... [object root]
+├── mcrobject
+│   └── Project
+│       └── doctype
+│           ├── 0000
+│           │   └── 01
+│           │       ├── Project_doctype_00000101
+│           │       │   └── ... [object root]
+│           │       ├── Project_doctype_00000109
+│           │       │   └── ... [object root]
+│           │       └── Project_doctype_00000110
+│           │           └── ... [object root]
+│           └── 1234
+│               └── 56
+│                   └── Project_doctype_12345678
+│                       └── ... [object root]
+├── mcrclass
+│   └── rfc5646
+│       └── ... [object root]
+├── mcruser
+│   └── editor1A@local
+│       └── ... [object root]
+├── mcracl
+│   └── rules
+│       └── ... [object root]
+└── mcrweb
+    └── pages
+        └── ... [object root]
+```
+
+## Offene Probleme
+
+  - Ein Objekt muss hart löschbar sein bisher ist nur 'soft'-löschen möglich.
+  - Nach löschen können alte Versionen immer noch aufgerufen werden
+  - Die Version muss bei `/receive/{ID}` als Attribut `?r=v{n}` mitgegeben werden, da sonst XSLT nichts von den Versionen weiß.
+
+## Fazit
```

Comment:
Den Abschnitt würde ich in der Doku weglassen. Das steht außerdem schon oben in der Einleitung.